src/YesSql.Core/Sql/SqlBuilder.cs

```diff
@@ -91,18 +91,20 @@ public virtual void InnerJoin(string table, string onTable, string onColumn, str
             {
                 toTable = toAlias;
             }
-            
+
             JoinSegments.Add(" INNER JOIN ");
-            JoinSegments.Add(_dialect.QuoteForTableName(_tablePrefix + table));
+            JoinSegments.Add(_dialect.SchemaNameQuotedPrefix() + _dialect.QuoteForTableName(_tablePrefix + table));
+
             if (!String.IsNullOrEmpty(alias))
             {
                 JoinSegments.AddRange(new[] { " AS ", _dialect.QuoteForTableName(alias) });
             }
+
```

Comment:
No schema here?

Reply:
Yes, there is probably an issue in here and on line 82. Will try to figure it out.